adafruit_displayio_sh1106.py

```diff
@@ -25,10 +25,13 @@
 
 # Support both 8.x.x and 9.x.x. Change when 8.x.x is discontinued as a stable release.
 try:
-    from fourwire import FourWire
+    from typing import Union
     from busdisplay import BusDisplay
+    from fourwire import FourWire
+    from i2cdisplaybus import I2CDisplayBus
 except ImportError:
     from displayio import FourWire
+    from displayio import I2CDisplay as I2CDisplayBus
     from displayio import Display as BusDisplay
```

Comment:
I think this will not work as intended.

On any version of CircuitPython, `from typing import Union` will fail, leading to the code under `except ImportError` being used in all cases.

It is necessary to use a separate try/except block for the "typing stuff" vs the stuff that adapts to different versions of CircuitPython, such as:

```py
try:
    from typing import Union
except ImportError:
    pass

try:
    from fourwire import FourWire
    from busdisplay import BusDisplay
    from i2cdisplaybus import I2CDisplayBus
except ImportError:
    from displayio import FourWire
    from displayio import I2CDisplay as I2CDisplayBus
    from displayio import Display as BusDisplay
```

Reply:
This was copied **directly** from the SSD1306 code -- https://github.com/adafruit/Adafruit_CircuitPython_DisplayIO_SSD1306/blob/40d7b8eb7a7d8fc91a62414d4f764dd48283d86d/adafruit_displayio_ssd1306.py#L34

This was tested on an QT Py RP2040 (PID: 4900) -- the SSD1306 driver code that the imports were copied from was tested on the same RP2040 **and** a Raspberry Pi Zero W.

That would imply that your statement of it failing on any version of CP seems inaccurate.

Test code can be found at #20